lib/action_dispatch/session/active_record_store_legacy.rb

```diff
@@ -0,0 +1,52 @@
+module ActionDispatch
+  module Session
+    module ActiveRecordStoreLegacy
```

Comment:
I feel like this would better be put within `ActionDispatch::Session::ActiveRecordStore` namespace/class. Maybe name it `ActiveRecordStore::LegacySupport` or something?